_drafts/SC3-1-1-html-lang-valid.md

```diff
@@ -0,0 +1,48 @@
+---
+name: SC3-1-1-html-lang-valid
+
+description: |
+ This rule checks the `xml:lang` and/or the `lang` attribute, and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute.
+
+### Expectation
+
+The value of the `lang` or `xml:lang` attribute conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+**Note**: If `lang` and `xml:lang` exists, then only `lang` is tested for conformance.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+This rule is only applicable in a scenario where assistive technologies are inconsistent about how they determine the language of the page.
```

Comment:
> There are known combinations of a popular operating system with browsers and assistive technologies that do not support the `lang` and `xml:lang` attributes.

Reply:
Did we want Wilco's text to completely replace the paragraph that is now in the Accessibility Support section? I can't see it from the comment, but I think that was what we talked about.